lib/api/openApiGenerator.ts

```diff
@@ -51,7 +51,8 @@ export function generateOpenApi(_request: KuzzleRequest): any {
       contact: {
         name: 'Kuzzle team',
         url: 'http://kuzzle.io',
-        email: 'hello@kuzzle.io'
+        email: 'hello@kuzzle.io',
+        discord: 'http://join.discord.kuzzle.io'
```

Comment:
Nitpicking: missing coma. Adding a coma at the end of the line prevent to have 2 lines in the git diff when you add a new item